src/example.pl

```diff
@@ -6,3 +6,9 @@
 foreach my $ele (@array) {
     print "$ele\n";
 }
+
+my @array2 = ("Nyan", "Cat");
+
+foreach my $ele (@array2) {
```

Comment:
**[perlcritic]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
**Module does not end with "1;"**, near <code>foreach my $ele (@array2) {</code>.<br>(Ref: [Modules::RequireEndWithOne](https://metacpan.org/pod/Perl::Critic::Policy::Modules::RequireEndWithOne))